app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def move_file(command: str) -> None:
+    com_ls = command.split()
+    path = com_ls[2]
+    sour_file = com_ls[1]
```

Comment:
The variable name `sour_file` is not very descriptive. Consider renaming it to `source_file` for better readability and to align with the checklist's suggestion for descriptive variable names.